plotly-behaviours.html

```diff
@@ -0,0 +1,244 @@
+<link rel="import" href="plotly-import.html">
+<script>
+    window.PlotlyBehaviours = window.PlotlyPlotlyBehaviours || {};
```

Comment:
Finally, this is just a single behavior, so it should be called PlotlyPlotBehavior.